### PR TITLE
Import component using atlassian-spring-scanner 2.x

### PR DESCRIPTION
**Pull request by mnowakowski-atlassian**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -204,7 +196,7 @@
         <jira.version>7.7.1</jira.version>
         <amps.version>6.3.15</amps.version>
         <plugin.testrunner.version>1.2.3</plugin.testrunner.version>
-        <atlassian.spring.scanner.version>1.2.13</atlassian.spring.scanner.version>
+        <atlassian.spring.scanner.version>2.1.3</atlassian.spring.scanner.version>
```
Comment (minaseem) [inline pom.xml:199]: while resolving dependencies, it says 
```
Dependency '''com.atlassian.plugin:atlassian-spring-scanner-annotation:2.1.3''' not found less... (⌘F1) 
Inspects a Maven model for resolution problems.
```

Comment (mnowakowski-atlassian) [reply pom.xml:199]: Do you have your maven configured accordingly?
https://developer.atlassian.com/server/framework/atlassian-sdk/working-with-maven/

Comment (minaseem) [reply pom.xml:199]: I have Atlassian SDK installed. Do I need to do some additional configuration? 
I went through the mentioned link. Not able to figure out what exactly I need to do.

Just to be clear please note that I am using intellijIdea, which is auto-importing package when I modify pom.xml, and  to run plugin I do `atlas-run`.

PS: I am new to maven settings. There are chances I might ask silly questions

Comment (mnowakowski-atlassian) [reply pom.xml:199]: Based on your error, it seems that your maven cannot find mentioned library. It suggests not configured maven repository.
For the sake of simplicity I'd suggest for now quit IDEA and try to run atlas-clean && atlas-run just from command line. This way you should have all necessary configuration provided, as per mentioned page.

Unfortunately I don't have capacity to assist you in configuring maven.

Comment (minaseem) [reply pom.xml:199]: @mnowakowski-atlassian  That worked. Thanks a lot. Really appreciate your continuous effort.

**Pull request merged**